### ISSUE TITLE
GTK warnings running docview sample

### ISSUE THREAD

**Issue by oneeyeman1** [wxWidgets/wxWidgets#24049]
### Description
Running the command:

./docview --mdi and trying to open all different documents one by one results in the following output on the console:

```
igor@WaylandGnome ~/wxWidgets/buildGTK/samples/docview $ ./docview --mdi

(docview:13897): Gtk-CRITICAL **: 22:24:11.208: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar

(docview:13897): Gtk-CRITICAL **: 22:24:15.168: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar

(docview:13897): Gtk-CRITICAL **: 22:24:15.169: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar

(docview:13897): Gtk-CRITICAL **: 22:24:18.661: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar
22:24:18: Debug: window wxMenuBar@0x55edfc962520 ("menubar") lost focus even though it didn't have it

(docview:13897): Gtk-CRITICAL **: 22:24:25.172: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar
22:24:25: Debug: window wxMenuBar@0x55edfc709120 ("menubar") lost focus even though it didn't have it
```

### Platform and version information

+ GTK version:  3.24.29
+ Which GDK  Wayland
+ Current theme:  default GNOME theme

**Comment by oneeyeman1** [comment]
Just cloned and build the git master of wxWidgets.

I see only those now:

[code]
igor@WaylandGnome ~/wxW/buildGTK/samples/docview $ ./docview --mdi

(docview:21726): Gtk-CRITICAL **: 23:09:10.395: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar

(docview:21726): Gtk-CRITICAL **: 23:09:18.082: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar

(docview:21726): Gtk-CRITICAL **: 23:09:18.082: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar

(docview:21726): Gtk-CRITICAL **: 23:09:24.910: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar

(docview:21726): Gtk-CRITICAL **: 23:09:30.237: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar
igor@WaylandGnome ~/wxW/buildGTK/samples/docview $ 

]/code]

**Comment by vadz** [comment]
This is same as #23499, please search the existing bug reports using the obvious keywords before opening new issues.

Also, please review your issues after submission, surely you would have noticed that your markup was  broken if you did?

**Comment by paulcor** [comment]
Actually, this is not the same as #23499. Should be fixed by 3af16cd063
